libs/designer/src/lib/core/state/templates/templateSlice.ts

```diff
@@ -35,6 +35,26 @@ export const templateSlice = createSlice({
         state.workflows[id].workflowName = name;
       }
     },
+    validateWorkflows: (state, action: PayloadAction<{ validateName: boolean; existingNames: string[] }>) => {
```

Comment:
so this is for basic tab validation (name and kind) only right - if so, could we consider naming it more specific?

This gives me the idea that this is validating the workflows in general including the other parts (connections, parameters)

Reply:
Shall I name it validateWorkflowsNameAndKind or validateWorkflowsBasicInfo (so we are clear this is validating the basic tab details)

Reply:
Yep I think that'd be ideal, I would prefer validateWorkflowsBasicInfo (in case of any further changes) since here we are aiming to validate everything in basic tab details